src/doc/src/reference/profiles.md

```diff
@@ -369,6 +369,9 @@ The output for each profile will be placed in a directory of the same name
 as the profile in the [`target` directory]. As in the example above, the
 output would go into the `target/release-lto` directory.
 
+NOTE: Custom profiles inherit package overrides as well and those take precedence
+over the default custom profile settings. See the [overrides](#overrides) section for more.
```

Comment:
What do you think about adding a sentence at line 250, instead of this?

```diff
- setting is not specified.
+ setting is not specified. Note that a custom profile inherits everything from
+ the other profile, including package and build [overrides](#overrides).
```